doc/source/conf.py

```diff
@@ -383,8 +383,8 @@ def setup(app):
         "https://fiona.readthedocs.io/en/stable/objects.inv",
     ),
     "folium": (
-        "https://python-visualization.github.io/folium/",
-        "https://python-visualization.github.io/folium/objects.inv",
+        "https://python-visualization.github.io/folium/latest",
+        "https://python-visualization.github.io/folium/latest/objects.inv",
```

Comment:
the objects.inv isn't present for the stable tag, isn't present at the redirect without a tag either.